businessRules/ideas.js

```diff
@@ -20,33 +27,21 @@ function sortedByDonationSumIdeas() {
 }
 
 function createIdeas(header, description, askedSum, userId) {
-    if (!header) {
-        throw new Error('Error. Please, check your header.');
-    }
+    validateHeader(header);
 
-    if (!description) {
-        throw new Error('Error. Please, check your description.');
-    }
+    validateDescription(description);
 
-    if (!askedSum) {
-        throw new Error('Error. Please, check your asked sum.');
-    }
+    validateAskedSum(askedSum);
 
-    if (!userId) {
-        throw new Error('Error. Please, check if you are logged in.');
-    }
+    validateUserId(userId);
 
     createIdea(header, description, askedSum, userId);
 }
 
 function updateIdeasStatus(ideaId, isApproved) {
     // Check if idea with this ideaId exists.
-    const ideas = getAllIdeas();
-    const idea = ideas.find((idea) => idea.id === ideaId);
+    findIdeaWithSameId(ideaId);
```

Comment:
The work `same` is confusing here. Why not simplify to `findIdea`?

Reply:
Renamed.